package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@netdata/dashboard",
-  "version": "2.19.9",
+  "version": "2.19.20",
```

Comment:
20? :)

Reply:
It's my lucky number 😆 I could swear the previous one was `2.19.19`